lib/lynis/lynistest.pm

```diff
@@ -259,14 +259,6 @@ sub compare_lynis_section_content {
         }
     }
 
-    # Filter out "System_Tools": "[2C- Starting dbus policy check...[28C"
-    # As this will likely change rather frequently and we have good mechanisms
-    # on our side to tackle this
-    if (grep(/Starting dbus policy check/, @section_current)) {
-        $result = "ok";
-        return $result;
-    }
-
```

Comment:
This dbus exception is approved by developer otherwise test case will fail randomly we have to pay efforts to review when fails.